### ISSUE TITLE
Trouble processing ABIDE II data (ndmg mode)

### ISSUE THREAD

**Issue by v715** [FCP-INDI/C-PAC#1008]
I am trying to use the latest Docker image for C-PAC to process MRI scans from the Autism Brain Imaging Data Exchange II [(ABIDE II)](http://fcon_1000.projects.nitrc.org/indi/abide/abide_II.html). I want to use the *ndmg* configuration to generate the connectomes, but am encountering errors that I don't know how to resolve. Any and all help is greatly appreciated :)

**Setting up the pipeline**

1. The ABIDE II data is not natively in the BIDS format, so I reformatted my files to look like this: <img width="338" alt="Screen Shot 2019-06-12 at 5 39 36 PM" src="https://user-images.githubusercontent.com/29757116/59388167-16ff6780-8d39-11e9-90f5-3d551a96b9d6.png">

2. I found a config file for using C-PAC in ndmg mode [here](https://github.com/FCP-INDI/C-PAC/blob/master/CPAC/resources/configs/pipeline_config_ndmg.yml) in your resources folder. I downloaded it to my local directory.

3. I then pulled the latest Docker Image for C-PAC with `docker pull fcpindi/c-pac:latest ` and started the pipeline with this command:
```
docker run -i --rm \
    -v <input_filepath>:/bids_dataset \
    -v <output_filepath>:/outputs \
    -v <config_filepath>:/configs \
    fcpindi/c-pac:latest /bids_dataset /outputs participant \
    --pipeline_file configs/ndmg.yml
```

**Output**

C-PAC runs for about 30 minutes and then returns this error message
```
190613-13:25:28,401 nipype.workflow INFO:


    Error of subject workflow resting_preproc_sub-30229_ses-1

    CPAC run error:

        Pipeline configuration: analysis_ndmg
        Subject workflow: resting_preproc_sub-30229_ses-1
        Elapsed run time (minutes): 30.2007092516
        Timing information saved in /outputs/log/cpac_individual_timing_analysis_ndmg.csv
        System time of start:      2019-06-13 12:55:19


190613-13:25:28,403 nipype.workflow INFO:
	 Removing working dir: /tmp/resting_preproc_sub-30229_ses-1
```

I have the latest crash log saved on my computer, but I don't know how to read the .pklz file.

**Comment by sgiavasis** [comment]
Hi @v715,

You can read the Nipype crash files using:
`nipypecli crash <path to crash file.pklz>`

Is there anything in the output stream or log file just before the end of the pipeline that mentions anything about an error or crash? Or, what the last thing was before it ended?

Steve

**Comment by sgiavasis** [comment]
Hi @v715 ,

Not sure if you've given this another try, but we just released C-PAC version 1.6.2, and this includes a fix for a bug some users were seeing with the ndmg-f preconfigured pipeline.

Try re-pulling fcpindi/c-pac:latest and see if you still get a crash.

Best,
Steve

**Comment by sgiavasis** [comment]
Closing due to inactivity- please feel free to reopen if you are still having issues.
